neuralcoref/train/conllparser.py

```diff
@@ -722,6 +726,7 @@ def _vocabulary_to_file(path, vocabulary):
     parser.add_argument('--n_jobs', type=int, default=1, help='Number of parallel jobs (default 1)')
     parser.add_argument('--gold_mentions', type=int, default=0, help='Use gold mentions (1) or not (0, default)')
     parser.add_argument('--blacklist', type=int, default=0, help='Use blacklist (1) or not (0, default)')
+    parser.add_argument('--spacy_model', type=str, default=None, help='spacys language model')
```

Comment:
`spacys` looks weird to me, I would put `spaCy's` or just simply call this "model name" as in other example scripts in spaCy. Note that this repo will be integrated more closely into spaCy in the future anyway.

Reply:
Changed to `model name`